crates/bevy_render/src/view/mod.rs

```diff
@@ -189,37 +191,39 @@ fn prepare_view_targets(
     let mut sampled_textures = HashMap::default();
     for (entity, camera) in &cameras {
         if let Some(target_size) = camera.physical_target_size {
-            if let Some(texture_view) = camera.target.get_texture_view(&windows, &images) {
-                let sampled_target = if msaa.samples > 1 {
-                    let sampled_texture = sampled_textures
-                        .entry(camera.target.clone())
-                        .or_insert_with(|| {
-                            texture_cache.get(
-                                &render_device,
-                                TextureDescriptor {
-                                    label: Some("sampled_color_attachment_texture"),
-                                    size: Extent3d {
-                                        width: target_size.x,
-                                        height: target_size.y,
-                                        depth_or_array_layers: 1,
-                                    },
-                                    mip_level_count: 1,
-                                    sample_count: msaa.samples,
-                                    dimension: TextureDimension::D2,
-                                    format: TextureFormat::bevy_default(),
-                                    usage: TextureUsages::RENDER_ATTACHMENT,
-                                },
-                            )
-                        });
-                    Some(sampled_texture.default_view.clone())
-                } else {
-                    None
-                };
-                commands.entity(entity).insert(ViewTarget {
-                    view: texture_view.clone(),
-                    sampled_target,
-                });
+            let texture_views = camera.target.get_texture_views(&windows, &images);
+            if texture_views.iter().filter_map(|x| *x).collect::<Vec<_>>().is_empty() {
+                continue;
             }
+            let sampled_target = if msaa.samples > 1 {
+                let sampled_texture = sampled_textures
+                    .entry(camera.target.clone())
```

Comment:
Thinking about this more do we need a resolve target per render target? I don't think so but knowledge here is a bit lacking.